frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -78,6 +78,7 @@
         vm.currentPhaseMetaAttributesVisibility = {};
         vm.phaseLeaderboardPublic = [];
         vm.currentPhaseLeaderboardPublic = false;
+        vm.approved_by_host = false;
```

Comment:
The variable `approved_by_host` does not make sense. Not all challenges need approval from host. Make this variable `eligible_to_submit` or something.